src/Select/Select.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+
+import Dropdown from "../Dropdown/Dropdown";
+import { InputTrigger } from "../Dropdown/DropdownTriggers";
+
+class Select extends React.Component {
+  constructor() {
+    super(...arguments);
+
+    this.state = {
+      value: this.getInitialValue()
+    };
+  }
+
+  buildItemsArray() {
+    const children = Array.isArray(this.props.children)
+      ? this.props.children
+      : [this.props.children];
+
+    return children.map(child => {
+      return {
+        html: child,
+        id: child.props.value,
+        selectedHtml: child.props.label,
+        selectable: !child.props.disabled
+      };
+    });
+  }
+
+  getInitialValue() {
+    if (!Array.isArray(this.props.children)) {
+      return this.props.children.props.value;
+    }
+    const child = this.props.children.find(
+      child => child.props.selected === true
+    );
+
+    return (child || this.props.children[0]).props.value;
+  }
+
+  handleInputChange(event) {
+    this.props.onChange(event);
+  }
+
+  handleDropdownChange(selectedOption) {
+    const event = new Event("input", { bubbles: true });
+
+    this.setState({ value: selectedOption.id });
+
+    this.input.value = selectedOption.id;
+    this.input.dispatchEvent(event);
+  }
+
+  render() {
+    return (
+      <div className={this.props.className}>
+        <input
+          className="dropdown-select input-value"
+          name={this.props.name}
+          ref={input => (this.input = input)}
+          style={{
+            display: "none"
+          }}
+          value={this.state.value}
+          onChange={this.handleInputChange.bind(this)}
+        />
+        <Dropdown
+          items={this.buildItemsArray()}
+          initialID={this.state.value}
+          onItemSelection={this.handleDropdownChange.bind(this)}
+          buttonClassName={"button dropdown-toggle"}
+          dropdownMenuClassName={"dropdown-menu"}
+          dropdownMenuListClassName={"dropdown-menu-list"}
+          dropdownMenuListItemClassName={"dropdown-menu-list-item"}
+          wrapperClassName={"dropdown"}
+          trigger={InputTrigger}
+          ref={dropdown => (this.dropdown = dropdown)}
+        />
+      </div>
+    );
+  }
+}
+
+Select.defaultProps = {
+  className: "dropdown-select",
+  onChange: undefined,
```

Comment:
There's no need to set to `undefined` as fallback value, because if not given it will already be `undefined`

Reply:
yeah, that one is basicly there to show the "default interface props"